src/UI/Status/Main.hs

```diff
@@ -104,6 +105,9 @@ instance WithContext (ListWithLength t e) where
 instance WithContext (E.Editor Text Name) where
   renderContext _ = str . show . cursorPosition . view E.editContentsL
 
+instance WithContext (FB.FileBrowser Name) where
+  renderContext _ _ = str "filebrowser"
```

Comment:
Hm.. just realizing this little bit. The file browser widget itself has a status bar. Will have to see what makes sense to display in our status bar.